third_party/lua/luadebug.lua

```diff
@@ -526,7 +526,11 @@ local function cmd_list_breakpoints()
     return false
 end
 
-local function cmd_down()
+--[[
+    Go up the stack. This advances toward the outermost frame,
+    to frames that have existed longer.
```

Comment:
What did you mean here?

Reply:
I'm surprised it's still worded not very clearly, and produce confusion. Could you please clarify what is unclear here - I'll try to rephrase it.

(FWIW - it was copied almost verbatim from gdb documentation)

Reply:
Pattern "advance toward smth, to smth" is hard to parse. I would rephrase it to 
```
It advances from the inner frame to the outer one that has existed longer.
```

Reply:
Well, thanks, I've almost applied it:
```
    Go up the stack. It advances from the callee frame to the caller.
```